### DISCUSSION TITLE
200.html fallback page for dynamic routes when using SSR?

### DISCUSSION THREAD

**Discussion by edwh** [nuxt/nuxt#17558] [General]
Static site generation used to produce a `200.html` page which you could configure as a fallback for dynamic routes, e.g. using nginx's `error_page`.

This page doesn't seem to get generated now.  Is it missing, or is there a different way to do this now?

**Comment by TheAlexLichter** [comment]
Still missing (follow https://github.com/nuxt/framework/issues/5003 along ☺️)
